nodes/nodes/eco-self-node-remediation-operator.adoc

```diff
@@ -27,4 +27,5 @@ include::modules/eco-self-node-remediation-operator-troubleshooting.adoc[levelof
 [id="additional-resources-self-node-remediation-operator-installation"]
 == Additional resources
 
-* The Self Node Remediation Operator is supported in a restricted network environment. For more information, see xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
\ No newline at end of file
+* The Self Node Remediation Operator is supported in a restricted network environment. For more information, see xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
+* xref:../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[Deleting Operators from a cluster]
```

Comment:
link doesn't work

Reply:
@prabinovRedhat Here too the links seem to be working fine. Did you access it from the preview links that I provided in the first comment of the PR?
http://file.rdu.redhat.com/avbhatt/bz-2109524/nodes/nodes/eco-self-node-remediation-operator.html#daemon-set-exists_self-node-remediation-operator-remediate-nodes

Reply:
works, thanks